packages/db/src/lib/retry.ts

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+import pRetry from 'p-retry';
+import Logger from '@cumulus/logger';
+
+type PromiseReturnType<T> = T extends (method: any) => infer R ? Promise<R> : never;
+
+export const RetryOnDbConnectionTerminateError = async <T>(
+  method: any,
+  retryConfig?: pRetry.Options,
+  log?: Logger): Promise<PromiseReturnType<T>> => {
+  const logger = log || new Logger({ sender: '@cumulus/db/retry' });
+  return await pRetry(
+    async () => {
+      try {
+        const result = await method;
+        return result;
+      } catch (error) {
+        if (error.message.includes('Connection terminated unexpectedly')) {
+          logger.error(`${error}. Retrying...`);
+          throw error;
+        }
+        throw new pRetry.AbortError(error);
+      }
+    },
+    {
+      ...retryConfig,
```

Comment:
put this after other parameters to override

Reply:
https://github.com/nasa/cumulus/pull/3332/commits/0962c4bd57ba8cc70c4df475c8fed21f1c06639e